### ISSUE TITLE
Accessibility statement: process issue

### ISSUE THREAD

**Issue by Hurstharrier** [best-practice-and-impact/aqua_book_revision#126]
From Chris P
On the accessibility page there is a pledge to 'consider your request and aim to get back to you within five working days.' - who will this responsibility fall to and how will requests be monitored?

**Comment by irisoren-sg** [comment]
@Hurstharrier to write to analysis function to establish what the statement should read

**Comment by Hurstharrier** [comment]
I have reviewed the Government Functional standard - it doesn't have an accessibility statement as in a PDF.

I will check with Phillipa about who is going to host the standard. My thought was for it to go onto  the Analysis Function page.

**Comment by Hurstharrier** [comment]
@zarbspace  @rebecca-wodcke-gad @irisoren-sg 
I have looked at all of the "coloured books" The only one in HTML in the public domain is the Green Book. It doesn't have an accessibility statement nor a route to feedback comments.

I think we should delete the statement - the HTML will have to comply with govt accessibility rules anyway and we are getting a hand with doing this.

The only thing is about feedback - we need a way of checking and responding to it if we ask for it. I think this is not an issue for producing the draft but one we can discuss afterwards with the wider group.

What do you think?

**Comment by rebecca-wodcke-gad** [comment]
remove accessibility part. ask wider AQuA book group about communication
